Portfolio.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h3>My Logo</h3>
+        <img class="tools" src="../Axel-Soler-Portfolio/images/Union.png" alt="">
```

Comment:
- [ ] Kindly wrap the hamburger in a `nav` tag
- [ ] Also, please make sure the hamburger is included in your design and the toolbar section matches the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/158703227-03f40dfd-36a5-4724-b51f-0c098ec6742a.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/158703165-d237905b-9f8d-410e-9570-9e8a5504ff45.png)